packages/rum-core/src/domain/configuration.spec.ts

```diff
@@ -91,6 +96,66 @@ describe('validateAndBuildRumConfiguration', () => {
     })
   })
 
+  describe('trackFrustrations', () => {
+    describe('without frustration-signals flag', () => {
+      it('defaults to false', () => {
+        expect(validateAndBuildRumConfiguration(DEFAULT_INIT_CONFIGURATION)!.trackFrustrations).toBeFalse()
+      })
+
+      it('the initialization parameter is ignored, no matter its value', () => {
+        expect(
+          validateAndBuildRumConfiguration({ ...DEFAULT_INIT_CONFIGURATION, trackFrustrations: true })!
+            .trackFrustrations
+        ).toBeFalse()
+        expect(
+          validateAndBuildRumConfiguration({ ...DEFAULT_INIT_CONFIGURATION, trackFrustrations: 'foo' as any })!
+            .trackFrustrations
+        ).toBeFalse()
+      })
+
+      it('does not impact "trackInteractions"', () => {
+        expect(
+          validateAndBuildRumConfiguration({ ...DEFAULT_INIT_CONFIGURATION, trackFrustrations: true })!
+            .trackInteractions
+        ).toBeFalse()
+      })
+    })
+
+    describe('with frustration-signals flag', () => {
+      beforeEach(() => {
+        updateExperimentalFeatures(['frustration-signals'])
+      })
+      afterEach(() => {
+        resetExperimentalFeatures()
+      })
+
+      it('defaults to false', () => {
+        expect(validateAndBuildRumConfiguration(DEFAULT_INIT_CONFIGURATION)!.trackFrustrations).toBeFalse()
+      })
+
+      it('the initialization parameter is set to provided value', () => {
+        expect(
+          validateAndBuildRumConfiguration({ ...DEFAULT_INIT_CONFIGURATION, trackFrustrations: true })!
+            .trackFrustrations
+        ).toBeTrue()
+      })
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏what about adding an expectation for `false` as well?